test/unit/integrations/klarna/klarna_helper_test.rb

```diff
@@ -74,7 +74,7 @@ def test_billing_fields
   def test_merchant_digest
     @helper = valid_helper
 
-    assert_field 'merchant_digest', "Efs6wr3Fhg4eGfQGMpelqTblLlXBzSCMz9fLUR9Bt74="
```

Comment:
This and line 10 had to change because `cart_items_payload` called `Money.from_amount(0.10).to_s` which returns 0.10. However the ActiveMerchant [only accepts cents](https://github.com/activemerchant/active_merchant/wiki/Patterns-&-Standards#amount) (or the other smallest available subunit for a currency) for amount. So the test was an incorrect example.